domains/aanthr0.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "name": "aanthr0",
+    "email": "anthromadayt@gmail.com"
+  },
+  "record": {
+    "URL: "na1.bluefoxhost.com:8215"
```

Json syntax issue
```suggestion
    "URL": "na1.bluefoxhost.com:8215"
```

If you wan't URL redirection it is better to specify the protocol as well

```suggestion
    "URL: "http://na1.bluefoxhost.com:8215"
```

---

Thank you for noticing this! I never would've seen it ¯\_(ツ)_/¯